packages/ariakit/src/ariakitStyles.css

```diff
@@ -1003,4 +1003,187 @@ responsible for the majority of the styling. */
 
 .bn-ak-link:where(.dark, .dark *) {
     color: hsl(204 100% 64%);
+}
+
+/* https://ariakit.org/components/hovercard */
+/*.bn-ak-button {*/
```

Comment:
Needed?

Reply:
Since these styles are copy/pasted from the ariakit docs, I think better to leave it in and commented out - makes it clearer why the styles aren't present if we have to update them